module/x/peggy/keeper/msg_server.go

```diff
@@ -478,3 +478,25 @@ func (k msgServer) LogicCallExecutedClaim(c context.Context, msg *types.MsgLogic
 
 	return &types.MsgLogicCallExecutedClaimResponse{}, nil
 }
+
+func (k msgServer) CancelSendToEth(c context.Context, msg *types.MsgCancelSendToEth) (*types.MsgCancelSendToEthResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	sender, err := sdk.AccAddressFromBech32(msg.Sender)
+	if err != nil {
+		return nil, err
+	}
+	err = k.RemoveFromOutgoingPoolAndRefund(ctx, msg.TransactionId, sender)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			sdk.EventTypeMessage,
+			sdk.NewAttribute(sdk.AttributeKeyModule, msg.Type()),
```

Comment:
the message type is not the module name, please use the correct Attribute. `AttributeKeyAction`

 ```suggestion
			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
						sdk.NewAttribute(sdk.AttributeKeyAction, msg.Type()),
```

This is done throughout the message server and will cause user confusion when searching for events.

Reply:
I'll open another pull request to fix it.

Reply:
if you have time to handle this it would be appreciated.